packages/shared/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "shared",
+  "version": "1.0.0",
+  "type": "module",
+  "engines": {
+    "node": "18.x.x",
+    "npm": "9.x.x"
+  },
+  "scripts": {
+    "lint:type": "npx tsc --noEmit",
+    "lint:js": "npx eslint \"src/**/*.ts\"",
```

```suggestion
    "lint:js": "npx eslint .",
```
let's link all files
add this to all eslint configs